java/src/main/java/com/logicalclocks/hsfs/StreamFeatureGroup.java

```diff
@@ -44,6 +60,11 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class StreamFeatureGroup extends FeatureGroupBase {
 
+  @Getter
+  @Setter
+  @JsonIgnore
+  private FeatureStore featureStore;
```

Comment:
Duplication between `FeatureGroupBase` and `StreamFeatureGroup`.

Reply:
Removed for now but it might bring back if tests fail